src/indieweb_utils/indieauth/flask.py

```diff
@@ -1,9 +1,27 @@
+from dataclasses import dataclass
+
 import requests
 
+@dataclass
+class IndieAuthResponse:
+    endpoint_response: dict
+
+
+class AuthenticationError(Exception):
+    pass
+
 
 def indieauth_callback_handler(
-    code, state, token_endpoint, code_verifier, session_state, me, callback_url, client_id, required_scopes
-):
+    code: str,
+    state: str,
+    token_endpoint: str,
+    code_verifier: str,
+    session_state: str,
+    me: str,
+    callback_url: str,
+    client_id: str,
+    required_scopes: str
```

Comment:
The docstrings indicate that required is a List of strings. 

```python
from typing import List

required_scopes: List[str]
```

What do you think about making this function kwarg only? i.e. It must be called like

```python
indieauth_callback_handler(code="hogehoge", state="fuga"...)
```

If you think that's a good idea, make the first argument a *

```python
def indieauth_callback_handler(
    *,
    code: str,
    ...
```